### DISCUSSION TITLE
2022/06/20/Spring%E7%9C%9F%E7%9A%84%E5%AE%8C%E5%85%A8%E8%A7%A3%E5%86%B3%E4%BA%86%E5%BE%AA%E7%8E%AF%E4%BE%9D%E8%B5%96%E5%90%97/

### DISCUSSION THREAD

**Discussion by giscus** [skitii/skitii#5] [General]
# 2022/06/20/Spring%E7%9C%9F%E7%9A%84%E5%AE%8C%E5%85%A8%E8%A7%A3%E5%86%B3%E4%BA%86%E5%BE%AA%E7%8E%AF%E4%BE%9D%E8%B5%96%E5%90%97/

Spring循环依赖竟然还有坑

https://skitii.vercel.app/2022/06/20/Spring%E7%9C%9F%E7%9A%84%E5%AE%8C%E5%85%A8%E8%A7%A3%E5%86%B3%E4%BA%86%E5%BE%AA%E7%8E%AF%E4%BE%9D%E8%B5%96%E5%90%97/

**Comment by AbnerHuang2** [comment]
不错

**Comment by AbnerHuang2** [comment]
sb

**Comment by AbnerHuang2** [comment]
评论加载完全看心情吗
